docs/UserGuide.md

```diff
@@ -16,171 +16,146 @@ By : `Team SE-EDU`  &nbsp;&nbsp;&nbsp;&nbsp; Since: `Jun 2016`  &nbsp;&nbsp;&nbs
-> * You can remove all the person's tags by typing `t/` without specifying any tags after it. 
+Finds task(s) whose names contain any of the given keywords.<br>
+Formats:
+> * `find <task>`
+> * `find <task>; <due date>` (`show tasks before the due date and also tasks without any due date`)
 
 Examples:
 
-* `edit 1 p/91234567 e/johndoe@yahoo.com`<br>
-  Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@yahoo.com` respectively.
+* `find project`
+* `find meeting; monday`
 
-* `edit 2 Betsy Crower t/`<br>
-  Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
+### 2.4. Listing all tasks : `list`
 
-### 2.5. Finding all persons containing any keyword in their name: `find`
+Shows a list of all tasks in the task manager.<br>
+Format:
+> * `list`
 
-Finds persons whose names contain any of the given keywords.<br>
-Format: `find KEYWORD [MORE_KEYWORDS]`
+### 2.5. Viewing help : `help`
 
-> * The search is case sensitive. e.g `hans` will not match `Hans`
-> * The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
-> * Only the name is searched.
-> * Only full words will be matched e.g. `Han` will not match `Hans`
-> * Persons matching at least one keyword will be returned (i.e. `OR` search).
-    e.g. `Hans` will match `Hans Bo`
+Formats:
+> * `help <command>` (provides specific information about the usage of the command)
+> * `help /all`
 
-Examples:
-
-* `find John`<br>
-  Returns `John Doe` but not `john`
-* `find Betsy Tim John`<br>
-  Returns Any person having names `Betsy`, `Tim`, or `John`
-
-### 2.6. Deleting a person : `delete`
+> Help is also shown if you enter an incorrect command e.g. `abcd`
 
-Deletes the specified person from the address book. Irreversible.<br>
-Format: `delete INDEX`
+### 2.6. Select a task : `select`
 
-> Deletes the person at the specified `INDEX`. <br>
-> The index refers to the index number shown in the most recent listing.<br>
-> The index **must be a positive integer** 1, 2, 3, ...
+Formats:
+> * `select <task>` (display all tasks with the same name for user to choose one)
+> * `unselect`
 
 Examples:
 
-* `list`<br>
-  `delete 2`<br>
-  Deletes the 2nd person in the address book.
-* `find Betsy`<br>
-  `delete 1`<br>
-  Deletes the 1st person in the results of the `find` command.
-
-### 2.7. Select a person : `select`
+* `select presentation`
 
-Selects the person identified by the index number used in the last person listing.<br>
-Format: `select INDEX`
+### 2.7. Update a task : `update`
 
-> Selects the person and loads the Google search page the person at the specified `INDEX`.<br>
-> The index refers to the index number shown in the most recent listing.<br>
-> The index **must be a positive integer** 1, 2, 3, ...
+Formats:
+> * `update <parameter> <new_value>` (when task to be updated has already been selected)
```

Comment:
update which task? how will the user specify which task to update?
how will the user know which task is being updated?